### ISSUE TITLE
Add SQN metric

### ISSUE THREAD

**Issue by twiecki** [quantopian/qrisk#9]
http://backtrader.readthedocs.io/en/latest/analyzers-reference.html#sqn

**Comment by ahgnaw** [comment]
Interesting metric, I've never seen it before. Input can be a list or dataframe of closed trades, what do you think?

**Comment by twiecki** [comment]
Yeah, specifying trades is actually not a straight-forward concept. In pyfolio we have `round_trip.py` for that which we could use to create it. But yeah, that list/dataframe as input here sounds good.
